spec/rubocop/cop/style/redundant_condition_spec.rb

```diff
@@ -292,6 +292,36 @@ def test
         RUBY
       end
 
+      it 'registers an offense and corrects when the branches contains method call with hash' do
```

Comment:
That's just my two cents. It will be symmetric with the `non-braced` of L310 :-)

```suggestion
      it 'registers an offense and corrects when the branches contains method call with braced hash' do
```